src/libraries/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -1,318 +1,51 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <EnableDefaultItems>true</EnableDefaultItems>
```

Comment:
Is the plan to do this for all of our libraries?  I'm wondering why System.Data.Common is special; most of our libraries are on the non-default items plan today.

Reply:
Great that you are asking as I'm preparing a change for that locally right now. I factored this one out because I felt that with the the three unused code files a separate PR is sensible. Many libraries (probably more than 70%) would benefit of default items being enabled. Give me a few hours to finish my change and open a PR and I will request your feedback then :)